features/track.yml

```diff
@@ -0,0 +1,53 @@
+name: Text tracks
+description: The `<track>` element is used as a child of the media elements that lets you specify a timed text track to be displayed in parallel with the media element.
+spec: https://html.spec.whatwg.org/multipage/media.html#timed-text-tracks
+group: media-elements
+compat_features:
+  - api.HTMLMediaElement.addTextTrack
+  - api.HTMLTrackElement
+  - api.HTMLTrackElement.cuechange_event
+  - api.HTMLTrackElement.default
+  - api.HTMLTrackElement.kind
+  - api.HTMLTrackElement.label
+  - api.HTMLTrackElement.readyState
+  - api.HTMLTrackElement.src
+  - api.HTMLTrackElement.srclang
+  - api.HTMLTrackElement.track
+  - api.TextTrack
+  - api.TextTrack.activeCues
+  - api.TextTrack.addCue
+  - api.TextTrack.cuechange_event
+  - api.TextTrack.cues
+  - api.TextTrack.id
+  # Excluded because of a non-real value
+  # - api.TextTrack.inBandMetadataTrackDispatchType
```

Comment:
@foolip - I think this is the right thing here, according to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/TextTrack/inBandMetadataTrackDispatchType) this is supported, but it has a limited value. If I don't comment out the line, the whole feature is not baseline because that is apparently not supported in Chrome.

Reply:
Does `npm run dist` throw an error if you include it? By non-real value we mean things like `true` in BCD that we can't handle and throw an error. I agree with excluding it, just want to understand.

To not have to fine tune the comment, you can just delete the lines. This whole block will go away if we move the tags to BCD.

Reply:
No, it does not throw an error.

Looking a bit deeper, this property it's not supported by Chrome, and there's [no plan to support it](https://issues.chromium.org/issues/41338648). 

I uncommented that line, and used `compute_from`, but I don't think that's the right answer either. Open to suggestions here. @ddbeck @captainbrosset